### PR TITLE
Onboard Validated Patterns documentation

### PR DESCRIPTION
**Pull request by yukinchan**

### CODE REVIEW

**Comment by Sau1506mya** [comment]
/pj-rehearse pull-ci-hybrid-cloud-patterns-docs-main-always-sucessful

File: ci-operator/config/hybrid-cloud-patterns/docs/hybrid-cloud-patterns-docs-main.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+build_root:
+  image_stream_tag:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+promotion:
```
Comment (ascerra) [inline ci-operator/config/hybrid-cloud-patterns/docs/hybrid-cloud-patterns-docs-main.yaml:6]: using promotion is meant to push an image into OpenShift CI's internal registry for post-submit jobs. I could be mistaking but I don't believe this is needed here.

**Pull request merged**